releases/v15_0.rst

```diff
@@ -0,0 +1,28 @@
+#######################
+LSST Stack 15.0 Release
+#######################
+
+Version 15.0 of the LSST Science Pipelines is now available. It is based on 2018 weekly 10.
+
+LSST Pipelines
+==============
+
+Full details of the LSST Science Pipelines 15.0 release can be found
+in the :ref:`extensive release notes <release-v15-0>`.  We have also
+published a `Characterization Metric Report <https://ls.st/DMTR-62>`_.
+
+Qserv and Data Access
+=====================
+
+The database team is on a separate release cadence.
```

Comment:
Let's not say this. Couple of reasons:

- We do not (formally) have a “database team”. We have a “Science Data Archive and Data Access Services” team. That's a mouthful, but if we're going to shorten it we should at least call it ”the Data Archive team” or something, rather than introducing an entirely new world.
- What does the database team produce? We might guess Qserv from the title, and, if we're lucky, we might even know what that means. But to the average science user, this is just speaking gibberish.